### PR TITLE
Added a client roster to the release rotation

### PR DESCRIPTION
**Pull request by rhuss**
Add client approvers to release rotation. Currently, only a subset of client approvers is added but could be extended to more (however I have not reached out to all of them yet)

@maximilien @navidshaikh If you are interested in being included in this list, please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,8 +63,6 @@ This roster is seeded with all approvers from Eventing workgroups.
 | v0.28   | 2021-12-14   | nak3           | vaikas          | 2021-11-03  | 2021-12-07 |
 | v0.29   | 2022-01-25   | markusthoemmes | matzew          | 2021-12-15  | 2022-01-18 |
 
-**NOTE:** v0.20 is moved by 3 weeks for end of year holidays
```
Comment (matzew) [inline README.md:59]: +1
I missed that :smile:

**Comment by matzew** [comment]
Feel free to extend the table (versions/dates) adding the new rosters!

**Comment by rhuss** [comment]
> Feel free to extend the table (versions/dates) adding the new rosters!

Sure, but not sure about the algo as its now three groups. I'd suggest continuing with

```
serving - client
eventing - client
serving - eventing
eventing - serving
....
```

(considering that the client roster only has 2 people on it right now ...)

**Comment by dprotaso** [comment]
Yeah it's not clear what the coordination would be - right now eventing & serving split their respective repos.

Would the client roster _own_ the `kn` client & plugin steps of the release?

**Comment by rhuss** [comment]
> Yeah it's not clear what the coordination would be - right now eventing & serving split their respective repos.
> 
> Would the client roster _own_ the `kn` client & plugin steps of the release?

Is it that only serving folks can release serving and eventing folks eventing ? Actually I would prefer more a roster that is aligned along time zones. I.e. one roster for APAC/EMEA and one for NA, and everybody should be able to release everything with an handover EOD of the first group. Wdyt ?

**Comment by dprotaso** [comment]
/lgtm

Let's land this and pull the last comment into an issue for a discussion

**Pull request merged**